code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -151,7 +150,7 @@
 	new /obj/item/clothing/accessory/leg_guard(src)
 	new /obj/item/clothing/accessory/arm_guard(src)
 	new /obj/item/clothing/head/helmet/hos(src)
-	new /obj/item/clothing/suit/storage/security/hos(src)
+	new /obj/item/clothing/suit/storage/security/officer/hos(src)
```

Comment:
As someone mentioned in the comments - this locker is not mapped into Horizon. Add the jacket to `/obj/structure/closet/secure_closet/hos`.